# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/moz.build
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/credentialmanagement/identity/moz.build
--- a/dom/credentialmanagement/identity/moz.build	Fri Jun 06 21:07:41 2025 +0300
+++ b/dom/credentialmanagement/identity/moz.build	Fri Jun 06 21:35:18 2025 +0300
@@ -13,22 +13,16 @@
     "IdentityCredentialSerializationHelpers.h",
     "IdentityNetworkHelpers.h",
     "NavigatorLogin.h",
-    "WebIdentityChild.h",
-    "WebIdentityHandler.h",
-    "WebIdentityParent.h",
 ]
 
 IPDL_SOURCES += [
-    "PWebIdentity.ipdl",
+    "IPCIdentityCredential.ipdlh",
 ]
 
 UNIFIED_SOURCES += [
     "IdentityCredential.cpp",
     "IdentityNetworkHelpers.cpp",
     "NavigatorLogin.cpp",
-    "WebIdentityChild.cpp",
-    "WebIdentityHandler.cpp",
-    "WebIdentityParent.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")